docs/advanced-chart-features/highcharts-typescript-declarations.md

```diff
@@ -9,6 +9,37 @@ fail-proofed development cycle for your Highcharts-based solutions.
 
 
 
+Summary of Changes from v7.2.1 to v8.0.0
+----------------------------------------
+
+With Highcharts v8 the file size of TypeScript declarations has been reduced by
+40 percent. This has been achieved by combining series options to more generic
+shared options. Following are the most important changes:
+
+* `AccessibilityAnnounceNewDataOptions` => `AccessibilityAnnounceNewDataOptionsObject`
```

Comment:
I think this looks slightly better as a table. Here's a quick comparison: 
![image](https://user-images.githubusercontent.com/7478772/90109322-ef949800-dd4b-11ea-899f-79d6dd28afb0.png)